### DISCUSSION TITLE
Adding smaller RFCs like RFC3325, RFC6050, RFC4412 and others

### DISCUSSION THREAD

**Discussion by JanHyteraMobilfunk** [sipsorcery-org/sipsorcery#654] [Ideas]
Hi Aaron,
are you interested in extensions of
RFC3325 ("P-Asserted-Identity"),
RFC6050 ("P-Asserted-Service"),
RFC4412 ("Resource-Priority"),
Probably RFC4028 ("Min-SE") and some otthers
for sipsorcery?
Currently they are accessible through the "UnkownHeaders" List of SIPHeader, but sometimes it is not that convenient.

Just let me know if you would like to have them. One of my colleagues or me will take care of the implementation.
Best regards,
Jan

**Comment by sipsorcery** [comment]
@JanHyteraMobilfunk defintiely! Any SIP RFC is a welcome additon to the code base.
